### PR TITLE
Add uncompressed and compressed message sizes and deprecate the old message size.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: api/src/main/java/io/opencensus/trace/NetworkEvent.java [hunk 1]
```diff
@@ -74,16 +74,33 @@ public static Builder builder(Type type, long messageId) {
...
+
+  /**
+   * Returns the compressed size in bytes of the {@code NetworkEvent}.
+   *
+   * @return the compressed size in bytes of the {@code NetworkEvent}.
+   */
+  public abstract long getCompressedMessageSize();
+
+  /**
+   * Use {@link #getUncompressedMessageSize}.
```
Comment (sebright) [inline api/src/main/java/io/opencensus/trace/NetworkEvent.java]: This should go in a `@deprecated` tag.

File: api/src/main/java/io/opencensus/trace/NetworkEvent.java [hunk 2]
```diff
@@ -103,12 +120,31 @@ public static Builder builder(Type type, long messageId) {
     public abstract Builder setKernelTimestamp(@Nullable Timestamp kernelTimestamp);
 
     /**
-     * Sets the message size.
+     * Use {@link #setUncompressedMessageSize}.
```
Comment (sebright) [inline api/src/main/java/io/opencensus/trace/NetworkEvent.java]: Same here

File: api/src/test/java/io/opencensus/trace/NetworkEventTest.java [hunk 3]
```diff
@@ -46,38 +46,52 @@ public void buildNetworkEvent_WithTimestamp() {
...
-            .setMessageSize(123L)
+            .setUncompressedMessageSize(123L)
+            .setCompressedMessageSize(456L)
             .build();
     assertThat(networkEvent.getKernelTimestamp()).isEqualTo(Timestamp.fromMillis(123456L));
     assertThat(networkEvent.getType()).isEqualTo(NetworkEvent.Type.RECV);
     assertThat(networkEvent.getMessageId()).isEqualTo(1L);
+    assertThat(networkEvent.getUncompressedMessageSize()).isEqualTo(123L);
+    // Test that getMessageSize returns same as getUncompressedMessageSize();
     assertThat(networkEvent.getMessageSize()).isEqualTo(123L);
```
Comment (sebright) [inline api/src/test/java/io/opencensus/trace/NetworkEventTest.java]: There should probably be a test for setMessageSize, too.

File: api/src/main/java/io/opencensus/trace/NetworkEvent.java (hunk 1 above)
Comment (bogdandrutu) [inline api/src/main/java/io/opencensus/trace/NetworkEvent.java]: Done.

File: api/src/main/java/io/opencensus/trace/NetworkEvent.java (hunk 2 above)
Comment (bogdandrutu) [inline api/src/main/java/io/opencensus/trace/NetworkEvent.java]: Done.

File: api/src/test/java/io/opencensus/trace/NetworkEventTest.java (hunk 3 above)
Comment (bogdandrutu) [inline api/src/test/java/io/opencensus/trace/NetworkEventTest.java]: Done.

**Pull request merged**